hpx/runtime/threads/policies/shared_priority_queue_scheduler.hpp

```diff
@@ -27,1327 +33,1333 @@
+                static std::string get_scheduler_name()
+                {
+                    return "shared_priority_queue_scheduler";
                 }
-            }
-
-            return time;
-        }
-#endif
 
-#ifdef HPX_HAVE_THREAD_STEALING_COUNTS
-        std::int64_t get_num_pending_misses(
-            std::size_t num_thread, bool reset) override
-        {
-            std::int64_t num_pending_misses = 0;
-
-            if (num_thread == std::size_t(-1))
-            {
-                for (std::size_t d = 0; d < num_domains_; ++d) {
-                    for (auto &queue : lp_queues_[d].queues_) {
-                        num_pending_misses += queue->get_num_pending_misses(
-                            reset);
+                // get/set scheduler mode
+                void set_scheduler_mode(scheduler_mode mode) override
+                {
+                    scheduler_base::set_scheduler_mode(mode);
+                    round_robin_ = mode & policies::assign_work_round_robin;
+                    steal_hp_first_ = mode & policies::steal_after_local;
+                    core_stealing_ = mode & policies::enable_stealing;
+                    numa_stealing_ = mode & policies::enable_stealing_numa;
+                    if (round_robin_)
+                    {
+                        spq_deb.debug(debug::str<>("set_scheduler_mode"),
+                            "assign_work_round_robin");
                     }
-
-                    for (auto &queue : np_queues_[d].queues_) {
-                        num_pending_misses += queue->get_num_pending_misses(
-                            reset);
+                    else
+                    {
+                        spq_deb.debug(debug::str<>("set_scheduler_mode"),
+                            "assign_work_thread_parent");
                     }
+                }
 
-                    for (auto &queue : hp_queues_[d].queues_) {
-                        num_pending_misses += queue->get_num_pending_misses(
-                            reset);
+                // ------------------------------------------------------------
+                void abort_all_suspended_threads() override
+                {
+                    // process all cores if -1 was sent in
+                    for (std::size_t d = 0; d < num_domains_; ++d)
+                    {
+                        numa_holder_[d].abort_all_suspended_threads();
                     }
                 }
 
-                return num_pending_misses;
-            }
-
-            std::size_t domain_num = d_lookup_[num_thread];
+                // ------------------------------------------------------------
+                // just function aliases
+                static constexpr auto& get_thread_number =
+                    threads::detail::get_thread_num_tss;
+                static constexpr auto& get_thread_numbers =
+                    threads::detail::get_thread_numbers_tss;
```

Comment:
not sure I understand what you are trying to say

Reply:
I'm just suggesting you use `using ...` instead of the function references as the only thing you gain with the reference is that you can remove `_tss`. Just my preference though.